Quarkus 2.7.5 problem

![78451649212946_ pic](https://user-images.githubusercontent.com/3773928/161887735-429eb2e1-9c86-4624-b668-52585aa50ef4.jpg)


After I upgrade my quarkus dependency from `2.7.0.Final` to `2.7.5.Final`, my project build process threw this error in above. How to fix it?

The Java version I used:

```bash
➤ java -version                                                                                                                                                     10:59:44
openjdk version "17.0.2" 2022-01-18
OpenJDK Runtime Environment (build 17.0.2+8-86)
OpenJDK 64-Bit Server VM (build 17.0.2+8-86, mixed mode, sharing)
```

Maven build command I used:

```bash
$ mvn install -DskipTests
```

Full error log:

```
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:2.7.5.Final:build (default) on project taskserver: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR]         [error]: Build step io.quarkus.deployment.logging.LoggingResourceProcessor#setupLoggingRuntimeInit threw an exception: java.lang.NullPointerException: Cannot invoke "io.quarkus.runtime.logging.InheritableLevel.isInherited()" because "inheritableLevel" is null
[ERROR]         at io.quarkus.runtime.logging.LoggingSetupRecorder.getLogLevel(LoggingSetupRecorder.java:300)
[ERROR]         at io.quarkus.runtime.logging.LoggingSetupRecorder.initializeBuildTimeLogging(LoggingSetupRecorder.java:265)
[ERROR]         at io.quarkus.deployment.logging.LoggingResourceProcessor.setupLoggingRuntimeInit(LoggingResourceProcessor.java:222)
[ERROR]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[ERROR]         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR]         at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[ERROR]         at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:882)
[ERROR]         at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
[ERROR]         at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR]         at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
[ERROR]         at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
[ERROR]         at java.base/java.lang.Thread.run(Thread.java:833)
[ERROR]         at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] -> [Help 1]
```

--- a/core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java
+++ b/core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java
@@ -295,7 +295,7 @@ public class LoggingSetupRecorder {
             T categoryConfig = categories.get(categoryName);
             if (categoryConfig != null) {
                 final InheritableLevel inheritableLevel = levelExtractor.apply(categoryConfig);
-                if (!inheritableLevel.isInherited()) {
+                if (inheritableLevel != null && !inheritableLevel.isInherited()) {
                     return inheritableLevel.getLevel();
                 }
             }

